happypose/pose_estimators/cosypose/cosypose/utils/cosypose_wrapper.py

```diff
@@ -32,62 +33,93 @@
         """
 
         self.dataset_name = dataset_name
         self.object_dataset = object_dataset
         self.detector, self.pose_predictor = self.get_model(
-            dataset_name, n_workers, renderer_type
+            dataset_name, model_type, n_workers, renderer_type
         )
 
-    def get_model(self, dataset_name, n_workers, renderer_type):
-        # load models
-        if dataset_name == "hope":
-            # HOPE setup
-            # python -m happypose.toolbox.utils.download --cosypose_models=detector-bop-hope-pbr--15246
-            # python -m happypose.toolbox.utils.download --cosypose_models=coarse-bop-hope-pbr--225203
-            # python -m happypose.toolbox.utils.download --cosypose_models=refiner-bop-hope-pbr--955392
-            detector_run_id = "detector-bop-hope-pbr--15246"
-            coarse_run_id = "coarse-bop-hope-pbr--225203"
-            refiner_run_id = "refiner-bop-hope-pbr--955392"
-        elif dataset_name == "tless":
-            # TLESS setup
-            # python -m happypose.toolbox.utils.download --cosypose_models=detector-bop-tless-pbr--873074
-            # python -m happypose.toolbox.utils.download --cosypose_models=coarse-bop-tless-pbr--506801
-            # python -m happypose.toolbox.utils.download --cosypose_models=refiner-bop-tless-pbr--233420
-            detector_run_id = "detector-bop-tless-pbr--873074"
-            coarse_run_id = "coarse-bop-tless-pbr--506801"
-            refiner_run_id = "refiner-bop-tless-pbr--233420"
-        elif dataset_name == "ycbv":
-            # YCBV setup
-            # python -m happypose.toolbox.utils.download --cosypose_models=detector-bop-ycbv-pbr--970850
-            # python -m happypose.toolbox.utils.download --cosypose_models=coarse-bop-ycbv-pbr--724183
-            # python -m happypose.toolbox.utils.download --cosypose_models=refiner-bop-ycbv-pbr--604090
-            detector_run_id = "detector-bop-ycbv-pbr--970850"
-            coarse_run_id = "coarse-bop-ycbv-pbr--724183"
-            refiner_run_id = "refiner-bop-ycbv-pbr--604090"
-        else:
-            msg = f"Not prepared for {dataset_name} dataset"
-            raise ValueError(msg)
-        detector = load_detector(detector_run_id, device)
+    def get_model(
+        self, dataset_name, model_type, n_workers, renderer_type
+    ) -> Tuple[Detector, PoseEstimator]:
+        """
+        Return CosyPose detector and pose estimator objects for a given dataset.
+
+        :param dataset_name: str, name of the dataset on which model was trained, hope|tless|ycbv
+        :param model_type: str, type of NN model (depends on training data), hope|tless|ycbv
+        :param renderer_type: str, which renderer to use, "panda3d" and "bullet" supported
+        :param n_workers: int, number of workers used in the renderer
+        :param model_type: str, what training data was used, "pbr" and "synth+real" supported
+        :return: tuple (Detector,PoseEstimator)
+        """
```

Comment:
Which docstring type is the dominating one in happypose?

Reply:
It looks like google with lines separating. I think we could stick to google. But maybe @nim65s could you relate to this topic?

Reply:
this is sphinx: https://www.sphinx-doc.org/en/master/usage/domains/python.html#info-field-lists

Reply:
@nim65s I meant the docstrigns in megapose @MedericFourmy sent link to. I know the one he used in this PR is sphinx, but the problem is that the rest of the codebase seems to be using google style